File: java/src/org/albanianskills/db/operations/HelloWorldOperation.java
package org.albanianskills.db.operations;

import org.albanianskills.db.models.Database;

public class HelloWorldOperation extends Operation {

    public HelloWorldOperation(Database db) {
        super("Hello World", db);
    }

    @Override
    public void execute() {
        System.out.println(db.helloWorld());
    }

    @Override
    public void revert() {
        System.out.println(db.helloWorld() + " reverted!");
    }
}

File: python/org/albanianskills/db/operations/hello_world_operation.py
from org.albanianskills.db.operations.operation import Operation


class HelloWorldOperation(Operation):

    def __init__(self, database):
        super().__init__("Hello World", database)

    def execute(self):
        print(self.database.hello_world())

    def revert(self):
        print(self.database.hello_world() + " reverted!")
        
